src/components/User/SignUpPage.js

```diff
@@ -39,49 +35,28 @@ export default function SignUpPage() {
 
     const handleOnSubmit = (e) => {
         e.preventDefault();
-        if (Object.keys(validationErrors).length > 0) {
-            toast.error('Please Enter Valid Values', { autoClose: 3000, pauseOnHover: false });
-            return;
-        }
         postHome();
     };
 
```

Comment:
Why you delete my work here?

Reply:
really? i didn't know i did something like that. sorry about that. let take a look at it